src/test/store/active-tab.test.js

```diff
@@ -99,20 +99,26 @@ describe('ActiveTab', function() {
 });
 
 describe('finalizeProfileView', function() {
-  function setup(search: string) {
+  function setup(search: string, withPages: boolean = true) {
```

Comment:
This setup functions tend to be more readable and maintainable when multiple options come in as an object. I would suggest changing the function signature from:

`setup('?view=active-tab&v=5', false);`

to:

`setup({search: '?view=active-tab&v=5', withPages: false});`

Or alternately you could inverse the `withPages` check to have this API:

`setup({search: '?view=active-tab&v=5', noPages: true});`
`setup({search: '?view=active-tab&v=5'});`